script/ProxyAdmin.s.sol

```diff
@@ -2,19 +2,15 @@
 pragma solidity 0.8.19;
 
 import {Script} from "forge-std/Script.sol";
+import {SphinxConfig} from './SphinxConfig.sol';
 import {QuestContractConstants as C} from "../contracts/libraries/QuestContractConstants.sol";
 
-contract ProxyAdminDeploy is Script {
-    function run() external {
-        uint256 deployerPrivateKey = vm.envUint("MAINNET_PRIVATE_KEY");
+contract ProxyAdminDeploy is Script, SphinxConfig {
+    function run() external sphinx {
         bytes memory data = vm.parseJsonBytes(vm.readFile("script/deployDataBytes.json"), "$.proxyAdminImpl");
 
-        vm.startBroadcast(deployerPrivateKey);
-
         // Deploy ProxyAdmin
         (bool success,) = C.DETERMINISTIC_DEPLOY_PROXY.call(data);
         require(success, "failed to deploy ProxyAdmin");
-
-        vm.stopBroadcast();
```

Comment:
Removed the broadcast code from this script because it's not necessary when deploying with Sphinx. Sphinx impersonates your deployer (i.e. your Gnosis Safe) in the `sphinx` modifier, which I've added to the `run()` function here.